src/styles/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($box-heigth, 1fr));
+
+  @media (min-width: $box-heigth * 4 ) {
+    grid-template-columns: repeat(3, minmax($box-heigth, 1fr));
+  }
+
+  @for $i from 1 through 6 {
+    &__box--#{$i} {
+      background-color: rgb((6 - $i) * 20%, 0, 0);
+    }
+  }
+
```

Comment:
You can replace this rule to the  **&__box** to keep sass-structure)

![image](https://user-images.githubusercontent.com/106536680/174485685-3f720415-a03c-4e2f-bd58-dca683baac6d.png)